js/adslot.jsx

```diff
@@ -1,75 +1,80 @@
-
 import React from 'react';
```

Comment:
That's just me deleting a blank line that the top of your file. The import statement is there.